tortoise/tortoise.go

```diff
@@ -506,30 +518,38 @@ func (t *turtle) runFull() (verified, changed types.LayerID) {
 	return verified, changed
 }
 
-func (t *turtle) computeEpochHeight(epoch types.EpochID) {
-	einfo := t.epoch(epoch)
+func (t *turtle) computeEpochHeight(lid types.LayerID) {
+	einfo := t.epoch(lid.GetEpoch())
 	heights := make([]uint64, 0, len(einfo.atxs))
 	for _, info := range einfo.atxs {
 		if !info.malfeasant {
 			heights = append(heights, info.height)
 		}
 	}
 	einfo.height = getMedian(heights)
+	t.logger.Debug(
+		"computed epoch height",
+		zap.Uint32("in layer", lid.GetEpoch().Uint32()),
+		zap.Uint32("for epoch", lid.GetEpoch().Uint32()),
+		zap.Uint64("height", einfo.height),
+	)
 }
 
 func (t *turtle) onBlock(header types.BlockHeader, data, valid bool) {
 	if header.LayerID <= t.evicted {
 		return
 	}
+
+	t.logger.Debug("on block", zap.Inline(&header), zap.Bool("data", data), zap.Bool("valid", valid))
```

Comment:
I find it confusing that this is called "data":
```go
type blockInfo struct {
    ...
    data bool // set to true if block is available locally
}
```
Nobody without good knowledge of the code will understand this log message and the meaning of `data` in general.
```suggestion
	t.logger.Debug("on block", zap.Inline(&header), zap.Bool("available", available), zap.Bool("valid", valid))
```

Reply:
i will keep it as data, as i refer to it as such accross the code. it is meant to be a debug log, i don't expect it to be clear to anyone without looking at code